test/integration/user_can_edit_item_quantity_in_cart_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+
+class UserCanEditItemQuantityInCart < ActionDispatch::IntegrationTest
+
+  test "user can update item quantity for cart items" do
+    item = create(:item)
+
+    visit items_path
+    click_link "Add to Duffel"
+    visit duffel_path
+
+    within '.cart-item-info' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.